WalletWasabi/Blockchain/TransactionBuilding/BuildTransactionResult.cs

```diff
@@ -20,7 +23,8 @@ public BuildTransactionResult(SmartTransaction transaction, PSBT psbt, bool spen
 			InnerWalletOutputs = innerWalletOutputs ?? new List<SmartCoin>();
 			SpentCoins = Guard.NotNullOrEmpty(nameof(spentCoins), spentCoins);
 		}
-
+		public BitcoinStore Store { get; }
```

Comment:
The concepts are mixed here IMO. BuildTransactionResult should contain relevant information already instead of extracting it later from BitcoinStore. How about storing a list of the input txs and remove code from here: 
https://github.com/molnard/WalletWasabi/blob/a0d1e195b26dafb428126dd904e5a32bfcbb1233/WalletWasabi.Gui/Controls/WalletExplorer/TransactionInfo.cs#L99